.github/workflows/build.yml

```diff
@@ -128,3 +128,171 @@ jobs:
 
       - name: Run make kotlin
         run: make -C lib/kotlin
+
+  lib-rust:
```

Comment:
this seems unrelated to what the pr title/description says it does? but since the jira ticket itself covers all languages I don't really see this as a problem :)

Reply:
yes it's actually based on:
- #2594 

but i think we can merge in one pass as well